public/js/poll/view.js

```diff
@@ -63,7 +63,7 @@
 		},
 		insertPoll: function(poll, callback) {
 			View.parsePoll(poll, function(html) {
-				$('#post-container .post-row[data-index="0"] .post-content').prepend(html);
+				$('.clearfix[data-index="0"] .content').append(html);
```

Comment:
This might be a tricky one, those id's are probably very old and no longer used.